### PR TITLE
Delay shutting down the PC

### PR DESCRIPTION
**Pull request by stefanroelofs**
Added 2 new options:
- New option to delay shutting down the Windows PC
- New option to send a message to the PC that is about to be shut down. If somebody is using the PC, you can warn them to save their work.

I tested the command locally, and the -t and -C arguments work as expected.

I've never editted a .bash file before, so this needs to be checked!

new variables are marked with a ? This should mean that they are optional (I don't want to break anyone's configuration.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @stefanroelofs, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#759).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by frenck** [comment]
I've requested a review from @pvizeli, I do not own any Windows machines myself, so that is hard to review.

File: rpc_shutdown/config.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
       {
         "alias": "test-pc",
         "address": "192.168.0.1",
-        "credentials": "user%password"
+        "credentials": "user%password",
+        "delay": "0",
```
Comment (pvizeli) [inline rpc_shutdown/config.json]: `"delay": 0`

File: rpc_shutdown/run.sh [hunk 2]
```diff
@@ -16,14 +16,21 @@ while read -r input; do
...
       
         # Not the correct alias
         if [ "$ALIAS" != "$input" ]; then
             continue
         fi
       
+	  # Check if delay is not empty
+        if [ "$DELAY" = "" ]; then
+            DELAY="0"
+        fi
```
Comment (pvizeli) [inline rpc_shutdown/run.sh]: Not needed if we have an default

**Comment by stefanroelofs** [comment]
I committed the requested changes.

I could also have a look at converting the script to Bashio? But maybe its better if this is a separate pull request.

**Comment by pvizeli** [comment]
@stefanroelofs would be nice, yeah

**Comment by pvizeli** [comment]
Arr, you are right, it needs a default. I was wrong. Sorry. Can you fix that again @stefanroelofs ? Thanks

**Comment by stefanroelofs** [comment]
no problem. This pull request is closed though. can you re-open? or do i start a new one?

**Comment by pvizeli** [comment]
You need start with a fresh PR/Branch

**Pull request merged**